dali/kernels/imgproc/resample/resampling_impl_cpu.h

```diff
@@ -44,20 +43,18 @@ void ResampleCol(Out *out, const In *in, int x, int w, const int32_t *in_columns
   if (static_channels < 0) {
     // we don't know how many channels we have at compile time - inner loop over filter
     for (int c = 0; c < channels; c++) {
-      float sum = bias;
+      float sum = 0;
       for (int k = 0; k < support; k++) {
         int srcx = x0 + k;
         if (clamp_left) if (srcx < 0) srcx = 0;
         if (clamp_right) if (srcx > w-1) srcx = w-1;
         sum += coeffs[k0 + k] * in[srcx * channels + c];
       }
-      out[channels * x + c] = clamp<Out>(sum);
+      out[channels * x + c] = ConvertSat<Out>(sum);
     }
   } else {
     // we know how many channels we have at compile time - inner loop over channels
-    float tmp[static_channels > 0 ? static_channels : 1];  // NOLINT
-    for (int c = 0; c < channels; c++)
-      tmp[c] = bias;
+    float tmp[static_channels > 0 ? static_channels : 1] = { 0 };  // NOLINT
```

Comment:
As in the GPU implementation.

Reply:
if the `if` was C++17 `if constexpr`, then yes.